src/test/components/TrackContextMenu.test.js

```diff
@@ -200,7 +200,8 @@ describe('timeline/TrackContextMenu', function() {
       expect(container.firstChild).toMatchSnapshot();
     });
 
-    it('network track will be hidden when a number is set for showTabOnly', () => {
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('network track will be hidden when a number is set for showTabOnly', () => {
```

Comment:
What's the plan here? I'm a little nervous leaving this in tree but skipped. I think we should either delete this code, or comment on why it's skipped with a TODO, and bonus points linking to an issue number. If these are turned back on in some local code you have then it's probably fine not to have an issue number.

Also, the same comment for other skipped tests in this patch.

Reply:
Also, what is the plan around hiding tracks in the active tab? Can we get away with not hiding things at all since we collapse resources? It seems like we should just delete the tests then.

Reply:
Yeah, I want to enable the skipped tests once I bring that functionality back, and they are inside my local patches. But you are right about this and the next one. We can actually remove these because they will be invalid. I thought I could change those something useful later but I can just create a new test instead.

Removed these two but kept the last one(the one with this title: `should use the finalizeActiveTabProfileView path and initialize active tab profile view state`) because this is a feature we want to test actually but currently it's not possible to test, because we are not initializing inside the `finalizeActiveTabProfileView`. Added a TODO for that one but didn't open an issue because we are going to enable it back on the global tracks PR.